### DISCUSSION TITLE
Fundamentos de Entity - como criar um projeto numa versão especifica?

### DISCUSSION THREAD

**Discussion by oimarcella** [balta-io/forum#591] [Tira Dúvidas] [answered]
Na aula ele menciona que podemos usar qualquer versão dentro da 5, se quisermos parecido com o exemplo da aula, porém na documentação não existe uma forma de passar essa configuração.
Eu pesquisei e vi que existe uma forma de fazer isso usando um arquivo global.json com a versão especificada. Essa é a abordagem correta?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @oimarcella, como vai?

Tem sim, de duas formas:
1- Na criação do projeto por exemplo (Só mencionar a versão usando o flag `-f`)
```
dotnet new console -o SeuApp -f net8.0
```

2- Manualmente após o projeto ser criado, alterando os arquivos .csproj, tanto a sessão `NET.Sdk` quanto a versão dos pacotes.

Para todos os casos é necessário ter instalado a versão do .NET que deseja usar. É possível instalar N versões do .NET na máquina lado a lado.

Minha recomendação: Utilize as mesmas versões do curso primeiro. Depois que terminar o curso e quiser variar, ai sim está livre para testar as versões mais novas.
